TestFiles/e2e.json

```diff
@@ -27,75 +27,120 @@
-  {"Path":"/api/actors/nm00000",      "PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":400,"jsonObject":[{"field":"status","value":400},{"field":"message","value":"Invalid Actor ID parameter"}]}},
-  {"Path":"/api/actors/nm12345",      "PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":404,"jsonObject":[{"field":"status","value":404},{"field":"message","value":"Actor Not Found"}]}},
-  {"Path":"/api/actors/nm0000173/foo","PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":404}},
-
-  {"Path":"/api/movies/ab12345",      "PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":400,"jsonObject":[{"field":"status","value":400},{"field":"message","value":"Invalid Movie ID parameter"}]}},
-  {"Path":"/api/movies/nm12345",      "PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":400,"jsonObject":[{"field":"status","value":400},{"field":"message","value":"Invalid Movie ID parameter"}]}},
-  {"Path":"/api/movies/TT12345",      "PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":400,"jsonObject":[{"field":"status","value":400},{"field":"message","value":"Invalid Movie ID parameter"}]}},
-  {"Path":"/api/movies/tt123",        "PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":400,"jsonObject":[{"field":"status","value":400},{"field":"message","value":"Invalid Movie ID parameter"}]}},
-  {"Path":"/api/movies/ttabcde",      "PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":400,"jsonObject":[{"field":"status","value":400},{"field":"message","value":"Invalid Movie ID parameter"}]}},
-  {"Path":"/api/movies/tt00000",      "PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":400,"jsonObject":[{"field":"status","value":400},{"field":"message","value":"Invalid Movie ID parameter"}]}},
-  {"Path":"/api/movies/tt12345",      "PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":404,"jsonObject":[{"field":"status","value":404},{"field":"message","value":"Movie Not Found"}]}},
-  {"Path":"/api/movies/tt0133093/foo","PerfTarget":{"Category": "DirectRead"},"Validation":{"StatusCode":404}},
-
-]
+
+  {"Path":"/api/actors?q=a","PerfTarget":{"Category":"SearchActors"},"Validation":{"StatusCode":400,"jsonObject":[{"field":"error","Validation":{"jsonObject":[{"field":"code","value":"BadArgument"},{"field":"message","value":"Invalid q (search) parameter"},{"field":"statusCode","value":400},{"field":"target","value":"q"},{"field":"innererror","Validation":{"jsonObject":[{"field":"code","value":"InvalidSearchParameter"},{"field":"maxLength","value":20},{"field":"minLength","value":2},{"field":"characterTypes","Validation":{"jsonArray":{"count":4,"byIndex":[{"index":0,"value":"lowerCase"},{"index":1,"value":"upperCase"},{"index":2,"value":"number"},{"index":3,"value":"symbol"}]}}}]}}]}}]}},
+
```

Comment:
why is there an extra lf between every line?

Reply:
easier to read each path statement